model_cards/English/toxicity.md

```diff
@@ -0,0 +1,91 @@
+
+
+## Overview
+
+![](https://github.com/conversationai/perspectiveapi/blob/lucy-model-card/model_cards/auc_wipd.png)
+The [ROC-AUC](https://en.wikipedia.org/wiki/Receiver_operating_characteristic) curve for the current TOXICITY model (TOXICITY@6). This shows True Positive Rate (y-axis) vs. False Positive Rate (x-axis).
+
+
+## Intended use
+
+
+#### Human-assisted moderation
+Make moderation easier with an ML assisted tool that helps prioritize comments for moderation, and create custom tasks for automated actions.
+&nbsp;
+
+#### Author feedback
+Assist authors in real-time when their comments might violate your community guidelines or be may be perceived as “Toxic” to the conversation. Use simple feedback tools when the assistant gets it wrong.
+&nbsp;
+
+#### Read better comments
+Organize comments on topics that are often difficult to discuss online. Build new tools that help people explore the conversation.
+&nbsp;
+
+
+## Uses to avoid
+
+#### Fully automated moderation
+Make moderation easier with an ML assisted tool that helps prioritize comments for moderation, and create custom tasks for automated actions.
+&nbsp;
+
+#### Character judgement
+This model only helps detect Toxicity in what a person said, and is not intended to detect anything about the individual who said it.
+&nbsp;
+
+
+
+## Model details
+
+#### Training data
+Proprietary from Perspective API, which includes comments from a online forums such as Wikipedia and New York Times, with crowdsourced labels of whether the comment is “toxic”, defined as “a rude, disrespectful, or unreasonable comment that is likely to make people leave a discussion”.
+
+#### Model architecture
+The model is a Convolutional Neural Network (CNN) trained with GloVe word embeddings.
+
+#### Values
+Community, Transparency, Inclusivity, Privacy, and Topic neutrality. Because of privacy considerations, the model does not take into account user history when making judgments about toxicity.
+&nbsp;
+
+
+## Evaluation data
+
+#### Overall evaluation data
+The overall evaluation result (shown above) is calculated using the held out test set associated with the training set for the specific model. Note that this means that each new model version is likely to have a different training and testing set, so overall results are not directly comparable across models.
+&nbsp;
+
+#### Unintended bias evaluation data
+The unintended bias evaluation result is calculated using a synthetically generated test set where a range of identity terms are swapped into template sentences, both toxic and non-toxic. Results are presented grouped by identity term. Note that this evaluation looks at only the identity terms present in the text, not the identities of comment authors or readers.

```

Comment:
Maybe clarify why not the identity of author or reader?

Reply:
Done